examples/fabric-admin/commands/interactive/InteractiveCommands.cpp

```diff
@@ -134,9 +161,13 @@ CHIP_ERROR InteractiveStartCommand::RunCommand()
         OpenLogFile(mLogFilePath.Value());
 
         // Redirect logs to the custom logging callback
-        chip::Logging::SetLogRedirectCallback(LoggingCallback);
+        Logging::SetLogRedirectCallback(LoggingCallback);
     }
 
+#if defined(PW_RPC_ENABLED)
```

Comment:
This flag is not define as 1 or 0, we can not enable this flag in Fabric Bridge since it will bring in some default server side RPC implementation, but Fabric Admin does not link those files so we don't have problem